BFAIR/normalization/normalization.py

```diff
@@ -237,30 +237,30 @@ def pqn_norm(
     element_name="Metabolite",
 ):
     """
-    Probabilistic Quotient Normalization: This method adjusts for dilutions.
+    Probabilistic Quotient Normalization : This method adjusts for dilutions.
     This is a modified version of Total Sum Intensity normalization
     Parameters
```

Comment:
Missing new line here ↑

Reply:
Oh I guess that got mixed up when I consolidated the new version in develop with this one... Thanks for spotting that!